content/en/docs/storage-providers/config.md

```diff
@@ -126,6 +126,12 @@ This section controls parameters for making storage and retrieval deals:
 The final value of `ExpectedSealDuration` should equal `(TIME_TO_SEAL_A_SECTOR + WaitDealsDelay) * 1.5`. This equation ensures that the miner does not commit to having the sector sealed too soon.
 {{< /alert >}}
 
+`StartEpochSealingBuffer` is minimum start epoch buffer to give time for sealing of sector with deal. In case a sector has multiple deal, this parameter enforces that sectors are packed for sealing/updating even if the nearest deal start date is close enough to the present even if there are more deals to be waited upon.
```

Comment:
I think a quick example would be really valuable here -- a hypothetical sector with 3 deals (with different start epochs), and indicate when they would start sealing.

Reply:
@arajasek Ack, I will add an example.

Reply:
Thank you!